### PR TITLE
[tests-only][full-ci] Add test for unlocking project space file

### PR DESCRIPTION
**Pull request by nabim777**
## Description
This PR adds the test for unlocking project space file

## Related Issue

- Part of: https://github.com/owncloud/ocis/issues/7625

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by nabim777** [comment]
This PR is block due to this issue https://github.com/owncloud/ocis/issues/7696

File: tests/acceptance/features/apiLocks/unlockFiles.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Feature: Unlock locked items
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/apiLocks/unlockFiles.feature:1]:
```suggestion
Feature: unlock locked items
```

File: tests/acceptance/features/apiLocks/unlockFiles.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  As a user
+  I want to unlock the resources previously locked by myself
+  So that other users can make changes to the resources
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+  @issue-7696
+  Scenario Outline: unlock a file in project space
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/apiLocks/unlockFiles.feature:10]:
```suggestion
  Scenario Outline: unlock a locked file in project space
```

File: tests/acceptance/features/apiLocks/unlockFiles.feature [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    Given using spaces DAV path
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "project-space" with content "some data" to "textfile.txt"
+    And user "Alice" has locked file "textfile.txt" inside space "project-space" setting the following properties
+      | lockscope | <lock-scope> |
+    When user "Alice" unlocks the last created lock of file "textfile.txt" inside space "project-space" using the WebDAV API
```
Comment (saw-jan) [inline tests/acceptance/features/apiLocks/unlockFiles.feature:17]: do we already have coverage for locking project space file?

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md [hunk 4]
```diff
@@ -149,5 +149,10 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 - [apiLocks/lockFiles.feature:180](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiLocks/lockFiles.feature#L180)
 - [apiLocks/lockFiles.feature:181](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiLocks/lockFiles.feature#L181)
 
+### [Error 500 when user try to unlock project space file](https://github.com/owncloud/ocis/issues/7696)
+
+- [apiLocks/unlockFiles.feature:21](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiLocks/unlockFiles.feature#L21)
+- [apiLocks/unlockFiles.feature:22](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiLocks/unlockFiles.feature#L22)
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:155]: @ScharfViktor do we wait for fix or just keep it here for now?

Comment (ScharfViktor) [reply tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:155]: let keep it here. I ask fix it on the next week

File: tests/acceptance/features/apiLocks/unlockFiles.feature (hunk 3 above)
Comment (nabim777) [reply tests/acceptance/features/apiLocks/unlockFiles.feature:17]: I try to search bu could not find so I think there is no coverage for locking and unlocking project spaces

Comment (saw-jan) [reply tests/acceptance/features/apiLocks/unlockFiles.feature:17]: Then I would suggest adding locking and unlocking tests

Comment (SwikritiT) [reply tests/acceptance/features/apiLocks/unlockFiles.feature:17]: there is some coverage for locking the project space file https://github.com/owncloud/ocis/blob/841ab73af20a70d0723eb5d67de65cbaab223afb/tests/acceptance/features/apiLocks/lockFiles.feature#L91

For the first batch of lock/unlock tests IMO we can shift the tests from core to ocis and provide the coverage for unlock as per there is coverage for locking and other edge cases can be added later.

File: tests/acceptance/features/apiLocks/unlockFiles.feature [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  I want to unlock the resources previously locked by myself
+  So that other users can make changes to the resources
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+  @issue-7696
+  Scenario Outline: unlock a locked file in project space
+    Given using spaces DAV path
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
```
Comment (SwikritiT) [inline tests/acceptance/features/apiLocks/unlockFiles.feature:12]:
```suggestion
    Given using spaces DAV path
    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
```

**Review by SwikritiT — APPROVED** [review]
LGTM

**Review by saw-jan — APPROVED** [review]
